springboard/views/core.py

```diff
@@ -93,9 +93,13 @@ def flat_page(self):
 
     @view_config(route_name='api_notify', renderer='json')
     def api_notify(self):
-        for working_dir, index_prefix in zip(self.all_repo_paths,
+        for working_dir, index_prefix in zip(self.all_repo_urls,
                                              self.all_index_prefixes):
-            fastforward.delay(os.path.abspath(working_dir), index_prefix)
+            if not any([
+                    working_dir.startswith('http://'),
+                    working_dir.startswith('https://')]):
+                fastforward.delay(os.path.abspath(working_dir), index_prefix)
+            # TODO: what is the best way to update a remote repo?
```

Comment:
It would be a POST to the repo in UCD `$ curl -XPOST http://localhost:6543/repos/unicore-sample-content.json`